src/core/ext/load_reporting/load_reporting_filter.c

```diff
@@ -156,37 +158,36 @@ static void init_channel_elem(grpc_exec_ctx *exec_ctx,
                               grpc_channel_element *elem,
                               grpc_channel_element_args *args) {
   GPR_ASSERT(!args->is_last);
-
   channel_data *chand = elem->channel_data;
   memset(chand, 0, sizeof(channel_data));
 
-  chand->id = (intptr_t)args->channel_stack;
+  const grpc_auth_context *auth_context =
```

Comment:
Assuming that this filter is server-side (which i'm pretty sure it is), you should grab the auth context from the `grpc_call_element_args` in `init_call_elem` like this:
```
grpc_server_security_context* server_context = args->context[GRPC_CONTEXT_SECURITY].value;
grpc_auth_context* auth_context = NULL;
if (server_context != NULL) {
  auth_context = server_context->auth_context;
}
  
```